### ISSUE TITLE
getDocuments with source

### ISSUE THREAD

**Issue by mobiletoly** [RxSwiftCommunity/RxFirebase#19]
Similar to this:
https://github.com/RxSwiftCommunity/RxFirebase/issues/16

we also need to add a function supporting source to query documents (instead of a single document). The one in

```
extension Reactive where Base: Query {
 ...
 // e.g.
public func getDocuments(source: FirestoreSource) -> Observable<QuerySnapshot> {
        return Observable<QuerySnapshot>.create { observer in
            self.base.getDocuments(source: source) { snapshot, error in
                if let error = error {
                    observer.onError(error)
                } else if let snapshot = snapshot {
                    observer.onNext(snapshot)
                }
                observer.onCompleted()
            }
            return Disposables.create()
        }
    }
}
```

I have a valid use case for that.

**Comment by arnauddorgans** [comment]
Thanks for your feeback, do you need anything else ? :)

**Comment by mobiletoly** [comment]
Alright, alright, I didn't mean to sound so demanding :) Thank you for your hard work!

**Comment by arnauddorgans** [comment]
Ahah sorry it was not mean, it was a real question 😂

**Comment by arnauddorgans** [comment]
If you need anything else i’ll add it to the next version :)

**Comment by mobiletoly** [comment]
thanks, I really appreciate it. I'm writing my first firebase-based project in iOS (used Firebase with Android before), and so far really enjoying RxFirebase.

**Comment by arnauddorgans** [comment]
Available in 0.3.7
